File: OLSource1/v1/VS_officedev/codesnippet/CSharp/walkthrough--changing-document-formatting-using-checkbox-controls_4.cs
        private void applyUnderlineFont_Click(object sender, System.EventArgs e)
        {
            if (this.applyUnderlineFont.Checked == true)
            {
                this.fontText.Underline = Word.WdUnderline.wdUnderlineSingle;
            }
            else
            {
                this.fontText.Underline = Word.WdUnderline.wdUnderlineNone;
            }
        }

File: OLSource1/v1/VS_officedev/codesnippet/VisualBasic/walkthrough--changing-document-formatting-using-checkbox-controls_4.vb
    Private Sub applyUnderlineFont_Click(ByVal sender As Object, ByVal e As System.EventArgs) _
        Handles applyUnderlineFont.Click

        If Me.applyUnderlineFont.Checked Then
            Me.fontText.Underline = Word.WdUnderline.wdUnderlineSingle
        Else
            Me.fontText.Underline = Word.WdUnderline.wdUnderlineNone
        End If
    End Sub